frontend/src/components/App.js

```diff
@@ -48,6 +49,8 @@ export default function App() {
             <Route path="/home">
               <Home />
             </Route>
+
+            <Redirect from="/" to="home" />
```

Comment:
Thanks for adding this. It makes the app start up more smoothly in its current state.